mentorship ios/Views/Home/SupportingViews/DetailListCell.swift

```diff
@@ -24,7 +24,12 @@ struct DetailListCell: View {
     //alert action,. To accept, delete, reject, or withdraw a request
     func alertAction() {
         guard let reqID = self.requestData.id else { return }
-        self.relationRequestActionAPI.actOnPendingRequest(action: self.actionType, reqID: reqID)
+        self.relationRequestActionAPI.actOnPendingRequest(action: self.actionType, reqID: reqID) { message in
```

Comment:
relationRequestActionAPI should be a protocol which you can inject with actual implementations.